File: 6KYU/Consonant value/solution.ts
export function solve(s: string) {
    let largest = 0;
    let current = 0;
    
    for(let i = 0; i < s.length; i++) {
        let c = s[i];
        if(c === 'a' || c === 'e' || c === 'i' || c === 'o' || c === 'u') {
          if(current > largest) {
            largest = current;
          }
          current = 0;
        } else {
          current += s[i].charCodeAt(0) - 96;
        }
    }
    
    return current > largest ? current : largest;
  }
  

File: 6KYU/Consonant value/solution.rb
def solve(s)
    current = 0
    largest = 0
    
    s.each_byte do |c|
      puts(c)
      if c == 97 || c == 101 || c == 105 || c == 111 || c == 117
        if current > largest
          largest = current
        end
        current = 0
      else
        current += (c - 96)
      end
    end
    return current > largest ? current : largest
end